<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>clientgui</class>
 <widget class="QMainWindow" name="clientgui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="twOptions">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>561</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabChat">
     <attribute name="title">
      <string>Chat</string>
     </attribute>
     <widget class="QPushButton" name="pbSend">
      <property name="geometry">
       <rect>
        <x>709</x>
        <y>486</y>
        <width>70</width>
        <height>43</height>
       </rect>
      </property>
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="leMessage">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>490</y>
        <width>701</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>-1</y>
        <width>151</width>
        <height>481</height>
       </rect>
      </property>
      <property name="title">
       <string>Users</string>
      </property>
      <widget class="QListWidget" name="qlwUsers">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>131</width>
         <height>441</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>601</width>
        <height>481</height>
       </rect>
      </property>
      <property name="title">
       <string>Chat Log</string>
      </property>
      <widget class="QTextEdit" name="teChatLog">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>581</width>
         <height>441</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tabOptions">
     <attribute name="title">
      <string>Options</string>
     </attribute>
     <widget class="QLineEdit" name="leUsername">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>261</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>absolution</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="leServerIP">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>60</y>
        <width>261</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>localhost</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="leServerPort">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>90</y>
        <width>261</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>8192</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Username:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>62</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Server IP:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Port:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pbConnect">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>160</y>
        <width>114</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="menuConnect"/>
    <addaction name="menuDisconnect"/>
    <addaction name="separator"/>
    <addaction name="menuQuit"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="menuConnect">
   <property name="text">
    <string>Connect...</string>
   </property>
  </action>
  <action name="menuDisconnect">
   <property name="text">
    <string>Disconnect...</string>
   </property>
  </action>
  <action name="actioQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="menuQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
